Apps/APN-UIKit/APN UIKit/APN UIKit/View/Customisation/CustomDataViewController.swift

```diff
@@ -46,13 +46,30 @@ class CustomDataViewController: UIViewController {
             propertyValueLabel.text = "Attribute Value"
         }
     }
+    
+    func isAllTextFieldsValid() {
+        if propertyValueTextField.isTextTrimEmpty ||
+            propertyValueTextField.isTextTrimEmpty ||
+            (source == .customEvents && eventNameTextField.isTextTrimEmpty) {
```

Comment:
```suggestion
        if propertyValueTextField.isTextTrimEmpty ||
           (source == .customEvents && eventNameTextField.isTextTrimEmpty) {
```

I think there is a duplicate `propertyValueTextField.isTextTrimEmpty`

Reply:
Fixed this in another PR.